+nosnoc/Options.m

```diff
@@ -0,0 +1,295 @@
+        time_freezing_reduced_model(1,1) logical = 0 % analytic reduction of lifter formulation, less algebraic variables (experimental)
+        time_freezing_hysteresis(1,1) logical = 0 % do not do automatic time freezing generation for hysteresis, it is not supported yet.
+        time_freezing_nonlinear_friction_cone(1,1) logical = 1 % 1 - use nonlienar friction cone, 0 - use polyhedral l_inf approximation.
+
+        time_freezing_quadrature_state(1,1) logical = 0 % make a nonsmooth quadrature state to integrate only if physical time is running
+        time_freezing_lift_forces(1,1) logical = 0 % replace \dot{v} = M(q)^{-1}f(q,v,u) by dot{v} = z,  M(q)z - f(q,v,u) = 0
+
+        % exerimental expert options
+        nonsmooth_switching_fun(1,1) logical = 0 % experimental: use c = max(c1,c2) insetad of c = [c1c2]
+        % expert mode: stabilize auxiliary dynamics in \nabla f_c(q) direction
+        stabilizing_q_dynamics(1,1) logical = 0
+        kappa_stabilizing_q_dynamics(1,1) double {mustBePositive} = 1e-5
+        % Verbose
+        print_level = 3
+
+        % Settings specific to CLS
+        friction_model (1,1) FrictionModel = FrictionModel.Conic; % use nonlinear friction ('Conic') or polyhedral approximation ('Polyhedral');
+        conic_model_switch_handling (1,1) ConicModelSwitchHandling = ConicModelSwitchHandling.Abs; % How to treat switch detection in v_t.
+        kappa_friction_reg (1,1) double {mustBeReal, mustBeNonnegative} = 0; % reg. term in friction equations to avoid large multipliers if no contact happens.
+        lift_velocity_state(1,1) logical = 0; % define auxliary algebraic vairable, dot{v} = z_v, to avoid symbolic inversion of the inertia matrix;
+        eps_cls double = 1e-3 % constraint: enforce f_c at Euler step with h * eps_cls
+
+
+        % Relaxation of terminal constraint
+        relax_terminal_constraint(1,1) {mustBeInteger} = 0; %  0  - hard constraint, 1 - ell_1 , 2  - ell_2 , 3 - ell_inf TODO enum
+        relax_terminal_constraint_from_above(1,1) logical = 0;
+        rho_terminal(1,1) double {mustBePositive} = 1e2;
+        relax_terminal_constraint_homotopy(1,1) logical = 0; % terminal penalty is governed by homotopy parameter
+
+        % Relaxation of terminal (or stage for equidistant grids) numerical/phyisical time constraints
+        relax_terminal_numerical_time(1,1) logical = 0; % instead of imposing \sum_h = T, add it as ell1_penalty term
+        rho_terminal_numerical_time(1,1) double {mustBeNonnegative} = 1e2
+        relax_terminal_numerical_time_homotopy (1,1) logical = 0; % us the homotopy parameter for the penalty
+        relax_terminal_physical_time(1,1) logical = 0; % instead of imposing \sum_h = T, add it as ell1_penalty term
+        rho_terminal_physical_time(1,1) double {mustBeNonnegative} = 1e2
+        relax_terminal_physical_time_homotopy (1,1) logical = 0; % us the homotopy parameter for the penalty
+
+        cost_integration(1,1) logical = 1
```

Comment:
what happens if this is false? does it make sense to have it

Reply:
This settings' name is horrible, We use it when we want to write an estimator so we don't integrate the cost but only take it at the discrete sample (stage) points. I am open to suggestions.

Reply:
Okay, it is fine, just adding the very same comment to the setting explaining it is sufficient.